src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.CSharp.Analyzers.targets

```diff
@@ -0,0 +1,29 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.CSharp.Analyzers.targets
```

Comment:
Most of the logic in this file is the same as in the VisualBasic file.  Should they just be merged into one file and have conditions on the `Analyzer` items?

Also, repeating my comment from earlier:

> In the analyzer layout under Microsoft.NET.Sdk/analyzers, there are cs and vb folders, and both have a Microsoft.CodeAnalysis.NetAnalyzers.dll file, as well as all its localized resources, which add up to close to 5 MB. Can we use a layout where this isn't duplicated across two folders for the different languages? Could we just put both the VB and C# analyzer DLLs in the same folder?

Reply:
reduced this to just deploy the dlls and not have duplicates. A about half of the size is for resource dlls which I believe we need. Let me know if there is something I am not thinking of that could reduce the size here.

Reply:
Yeah I was saying that we should try not to duplicate the same resources across C# and VB, which is what I think as happening.  We do still need them once though :-)